package.json

```diff
@@ -6,17 +6,25 @@
   "scripts": {
     "codegen": "graph codegen",
     "codegen:sushiswap": "graph codegen subgraphs/sushiswap.yaml",
+    "codegen:bar": "graph codegen subgraphs/bar.yaml",
     "build": "graph build",
     "build:sushiswap": "graph build subgraphs/sushiswap.yaml",
+    "build:bar": "graph build subgraphs/sushiswap.yaml",
     "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/",
     "deploy:sushiswap": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ sushiswap/sushiswap subgraphs/sushiswap.yaml",
```

Comment:
Add this script: `"deploy:bar": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ sushiswap/sushi-bar subgraphs/bar.yaml"`